autosklearn/ensemble_builder.py

```diff
@@ -319,6 +332,42 @@ def main(self, return_pred=False):
         if return_pred:
             return valid_pred, test_pred
 
+    def get_worst_disk_consumption(self):
```

Comment:
Hey, I'm wondering whether we could just read the required disk space of a model/predictions when actually scoring the model and then keep that information within the large dictionary containing all kinds of information about the models?